stanford-earth/matson
# Earth 687 css fixes

# READY FOR REVIEW

# Summary
- Added in fixes mostly for accessibility and to accommodate WYSIWYG in various PTs.

# Needed By (Date)
- No urgency

# Urgency
- Not very.

# Steps to Test
FIRST: disable the CSS injector rules Fixes for Barbara" and "Text Re-Styling for Consistency and Accessibility" THEN
1. Add a header, list, or link to various PTs (for example, Deep Linking Hero banner or Links Banner). Check to make sure font is good color/is visible.
3. Next, add an h2 to a plain WYSIWYG field. Note that it has the red dash.
4. Go to the WYSIWYG and add one of the other styles (Teal H2) and verify that it looks OK.
5. Check paragraph padding on Earth Matters news items and verify that it looks OK.
6. Check paragraph padding on WYSIWYG PT and verify that it looks OK.
7. Look at social actions area on a news item. Note that there is a top and bottom border.


# Affected Projects or Products
- When this is pushed to production, the CSS injector rules "Fixes for Barbara" and "Text Re-Styling for Consistency and Accessibility" should be removed.

# Associated Issues and/or People
- Also related to EARTH-566 and EARTH-557

# See Also
- [PR Checklist](https://gist.github.com/sherakama/0ba17601381e3adbe0cad566ad4d80a5)
